.github/ISSUE_TEMPLATE/ql--false-positive.md

```diff
@@ -0,0 +1,36 @@
+---
+name: CodeQL False positive
+about: Report CodeQL alerts that you think should not have been detected (not applicable, not exploitable, etc.)
+title: False positive
+labels: false-positive
+assignees: ''
```

Comment:
Do we want to assign this to the first responder automatically? @github/codeql-first-responder

Reply:
I don't think that will work - we can't assign teams to issues, only individuals. If we put the handle in the issue template body, the issue creator would need to be a member of our org to actually notify a team in our org.

Reply:
Ah...didn't know that. I am happy with this template, though. @turbo should probably approve since he had some suggestions earlier.